render/src/simple_element.rs

```diff
@@ -20,9 +20,16 @@ fn write_attributes<'a, W: Write>(maybe_attributes: Attributes<'a>, writer: &mut
         None => Ok(()),
         Some(mut attributes) => {
             for (key, value) in attributes.drain() {
-                write!(writer, " {}=\"", key)?;
-                escape_html(&value, writer)?;
-                write!(writer, "\"")?;
+                if key.chars().nth(0).unwrap_or('.') == 'b' && key.chars().nth(1).unwrap_or('.') == '!' {
```

Comment:
Not my favorite addition as its a bit of a hack-- definitely want a better way to do this in the future. This interprets b! at the start of an attribute as a boolean attribute in HTML, which will cause it to optionally render the key based on a value of "true". 

Mostly because we need _some way_ to do this today as things like selected='false' in an option isn't actually valid html.